components/ResultsPageComponents/StaffSuggestions/StaffSuggestions.jsx

```diff
@@ -0,0 +1,37 @@
+import { Text, Flex, Image } from "@chakra-ui/react";
+
+export default function StaffSuggestions() {
+  return (
+    <Flex
+      minWidth="350px"
+      maxWidth="1100px"
+      alignSelf={["center", "start"]}
+      align={["center", "flex-start"]}
+      flex={[1, 2]}
+      direction="column"
+      gap={["10px", "20px"]}
+      px={["5px", "50px"]}
+      py={["25px", "50px"]}
+      justify="center"
+    >
+      <Flex direction="column">
```

Comment:
Question about nesting: we have a large Flex container with a lot of attributes, why do we need another Flex container inside it?

Reply:
Line 5 then line 17

Reply:
that's a really good point @OlaShabalina 👍🏽 
good spot 👍🏽 

I removed the nested Flex container on line 17, and it made no difference to the layout in my local environment.

What do you think @RCode-Blue ?

Reply:
Yes, please remove the nested Flex.

Sorry @OlaShabalina you're right, it was unnecessary.